aws/createResources.sh

```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env bash
 
 ROOT_NAME=DevDay2
-BUCKET_NAME=budilov-$ROOT_NAME
+# Bucket name must be all lowercase, and start/end with lowecase letter or number
+BUCKET_NAME=budilov-${ROOT_NAME,,}
```

Comment:
`${stringname,,}` (double comma's) will lowercase the $stringname variable. ^^ will uppercase.

Note: Newer versions of bash needed to support Unicode. [StackExchange question in it](http://askubuntu.com/questions/383352/command-to-convert-an-upper-case-string-to-lower-case).